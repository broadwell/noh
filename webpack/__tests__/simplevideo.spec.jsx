import React from 'react';
import { shallow } from 'enzyme';
import SimpleVideo from '../components/SimpleVideo';

describe('<SimpleVideo>', () => {
  it('renders as expected', () => {
    const component = shallow(<SimpleVideo />);
    expect(component).toMatchSnapshot();
  });
});
