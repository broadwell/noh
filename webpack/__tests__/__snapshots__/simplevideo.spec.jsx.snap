// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SimpleVideo> renders as expected 1`] = `
<video
  controls={true}
  id="player"
  src=""
/>
`;
